lumicks/pylake/population/dwelltime.py

```diff
@@ -723,6 +784,10 @@ def hist(
             scaling for the x-axis; when `None` default is "linear"
         yscale : {"log", "linear", None}
             scaling for the y-axis; when `None` default is same as `bin_spacing`
+        int_steps : int
+            To obtain accurate pdf values when plotting, we sample the pdf at a finer resolution
+            than the histogram bins. The number of steps we evaluate per bin is defined as:
+            max(3, int_steps // n_bins). Default: 10000.
```

Comment:
do we want to expose this to the users? I think it'd actually be more confusing (because most people won't notice this weird mixed observation range stuff). I would say let's just give it a default for us internally and if we hear of issues we can expose it then.